lib/model/Capability.mjs

```diff
@@ -124,13 +124,13 @@ const namePerType = {
   EffectParameter: cap => `${cap.comment || `Effect parameter`} ${startEndToString(cap.parameter)}`,
   SoundSensitivity: cap => appendInBrackets(`Sound sensitivity ${startEndToString(cap.soundSensitivity)}`, cap.comment),
   GoboIndex: cap => {
-    let name = `Gobo `;
+    let name = ``;
 
     if (cap.hasComment) {
-      name += cap.comment.replace(/^gobo(?: shake)?\s*/i, ``);
+      name = cap.comment.replace(/^(?:gobo\s+|)(?!open)/i, `Gobo `);
```

Comment:
The RegEx should also include `Closed`, i.e. `/^(?:gobo\s+|)(?!(open|closed))/i`